core/src/main/java/org/elasticsearch/index/query/QueryShardContext.java

```diff
@@ -115,18 +114,12 @@ public QueryShardContext(QueryShardContext source) {
 
     public void reset() {
```

Comment:
I see that I had already asked this before, ignore me, you already said we'll do it in another PR :)